src/assets/scripts/app.js

```diff
@@ -0,0 +1,7 @@
+const menuButton = document.getElementById("navigation-toggle");
+menuButton.addEventListener("click", () => {
+    const ariaExpanded = menuButton.ariaExpanded === "false";
+    menuButton.ariaExpanded = ariaExpanded;
+    const menu = document.getElementById("navigation-menu");
+    menu.classList.toggle("visually-hidden");
```

Comment:
There's a more robust way of doing this that also doesn't require you to override the `.visually-hidden` class properties on larger screen sizes. Let's discuss.

Reply:
```css
.navigation__link:has([aria-expanded="true"]) + .navigation__menu {
  display: block;
}

.navigation__link:has([aria-expanded="false"]) + .navigation__menu {
  display: none;
}

@media (min-width: 600px) {
  .navigation__link:has([aria-expanded="false"]) + .navigation__menu {
    display: block;
  }
}
```